formulaire/src/main/resources/public/ts/directives/question/question-item/question-item.directive.ts

```diff
@@ -128,6 +137,12 @@ class Controller implements IViewModel {
         let nextElements: FormElement[] = this.formElements.all.filter((e: FormElement) => e.position === this.getPosition() + 1);
         return nextElements.length > 0 ? nextElements[0].id : null;
     }
+
+    isImageMissingLabel = (): boolean => {
+        const choices = this.question.choices.all;
```

Comment:
je pense que tu n'as pas besoin de const, tu peux soit mettre let ou alors ne pas faire de variable et directement mettre this.question.choices.all.some(...)